css/alx-dark.css

```diff
@@ -884,16 +884,24 @@ body.dark .panel .list-group-item.in_second_deadline {
 body.dark .panel .list-group-item.in_second_deadline a {
   color: var(--col-second-deadline-link) !important;
 }
+
+body.dark .panel .list-group-item.in_second_deadline a:hover {
+  color: var(--col-light-400) !important;
+}
 /* Dashboard styles ends here */
 
 /* Pending logs styles starts here */
```

Comment:
And also change the description from Pending logs to something like second deadline... we'd easily understand that

Reply:
I have implemented this also